modules/kms/README.md

```diff
@@ -70,18 +54,34 @@ module "kms" {
     }
   }
 }
-# tftest modules=1 resources=6 inventory=basic.yaml
+# tftest modules=1 resources=6 inventory=basic.yaml e2e
+```
+
+### Using an existing keyring
+
+```hcl
+module "kms" {
+  source     = "./fabric/modules/kms"
+  project_id = var.project_id
+  iam = {
+    "roles/cloudkms.admin" = ["group:${var.group_email}"]
+  }
+  keyring        = { location = var.region, name = "test-1" }
```

Comment:
In the [setup module](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/tests/examples_e2e/setup_module/main.tf#L123) we are creating a `keyring` but not exporting its name / id as variable.

This test fails, as it assumes existence of "test-1" keyring which will not exists in test environment.

Instead, please modify setup_module to export keyring id as variable ([here](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/tests/examples_e2e/setup_module/e2e_tests.tfvars.tftpl) and [here](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/tests/examples/variables.tf) and then use the variable here.

Reply:
Thanks for the suggestion Wiktor. Exported the existing keyring.